### PR TITLE
Multiple events in one day

### PR DESCRIPTION
**Pull request by daeglin**

### CODE REVIEW

File: frontend/src/app/services/events.ts [hunk 1]
```diff
@@ -76,26 +79,30 @@ export class EventHandler {
...
-    const active = shuffle(currentState.triggerStates.filter(ts =>
+    const active = currentState.triggerStates.filter(ts =>
       ts.activeBefore === undefined
-        || ts.trigger.reactivateAfter !== undefined && ts.activeBefore >= ts.trigger.reactivateAfter));
-    const triggerState = active.find(ts => ts.trigger.condition(eventInput));
+        || ts.trigger.reactivateAfter !== undefined && ts.activeBefore >= ts.trigger.reactivateAfter);
+    const triggered = active.filter(ts => ts.trigger.condition(eventInput));
 
-    if (!triggerState) return;
+    if (!triggered) return;
```
Comment (jnwltr) [inline frontend/src/app/services/events.ts:95]: Always false since `![] === false`, check for `.length` if empty array makes a difference.

**Review by jnwltr — CHANGES_REQUESTED** [review]
I am sorry, I do not understand it here. Maybe we could check it together. Where do you cycle through the events to display them all? I would put this logic to `resumeEvent` method.

File: frontend/src/app/game/events-layout/events-layout.component.ts [hunk 2]
```diff
@@ -27,7 +27,9 @@ export class EventsLayoutComponent {
       }
     }
 
-    this.gameService.event = undefined;
-    this.gameService.setSpeed('play');
+    this.gameService.currentEvent = this.gameService.events.shift();
```
Comment (daeglin) [inline frontend/src/app/game/events-layout/events-layout.component.ts:30]: This is where we take another event from the event queue

File: frontend/src/app/services/events.ts (hunk 1 above)
Comment (daeglin) [reply frontend/src/app/services/events.ts]: My bad. After several mistakes like this, I am finally learning.

File: frontend/src/app/game/events-layout/events-layout.component.ts (hunk 2 above)
Comment (jnwltr) [reply frontend/src/app/game/events-layout/events-layout.component.ts:30]: Sorry, I was searching by the hidden method name.

**Review by jnwltr — APPROVED** [review]

**Pull request merged**